lib/evaluate/call.h

```diff
@@ -111,6 +111,11 @@ class ActualArgument {
   void set_isPassedObject(bool yes = true) { isPassedObject_ = yes; }
 
   bool Matches(const characteristics::DummyArgument &) const;
+  common::Intent dummyIntent() const { return dummyIntent_; }
+  ActualArgument &set_dummyIntent(common::Intent intent) {
```

Comment:
why mix camelCase and underscores?

Reply:
I see other instances of this. Is this done intentionally?

Reply:
`set_...` signifies a mutator.

Reply:
The data member is called "dummyIntent_" and I wanted a mutator for it.  Our style guide says that mutators should start with "set_" followed by the name of the data member without the underscore.  The result is this mix of camelCase and underscores.

Reply:
Got it.